src/tests/ftest/util/dmg_utils_base.py

```diff
@@ -685,7 +685,7 @@ def __init__(self):
                     super().__init__("/run/dmg/storage/replace/nvme/*", "nvme")
                     self.old_uuid = FormattedParameter("--old-uuid {}", None)
                     self.new_uuid = FormattedParameter("--new-uuid {}", None)
-                    self.no_reint = FormattedParameter("--no-reint", False)
+                    self.host = FormattedParameter("--host {}", None)
```

Comment:
Need to update dmg args for the telemetry query as well? I'd suggest running tests for the telemetry feature to ensure nothing was broken.

Reply:
Was this addressed?

Reply:
yes and telemetry feature is being run in CI